src/components/organisms/ArrowUpOrganism/ArrowUpOrganism.jsx

```diff
@@ -1,15 +1,17 @@
+import { forwardRef } from 'react';
+
 import { smoothScrollToTop } from '../../../helpers/smoothScroll';
 import { ArrowUpMolecule } from '../../molecules/ArrowUpMolecule/ArrowUpMolecule';
 
 import css from './ArrowUpOrganism.module.css';
 
-function ArrowUpOrganism() {
+const ArrowUpOrganism = forwardRef((props, ref) => {
   const handleClick = () => smoothScrollToTop();
   return (
-    <div className={css['button__arrow--up']} onClick={handleClick}>
+    <div ref={ref} className={css['button__arrow--up']} onClick={handleClick}>
```

Comment:
You don't need to create  `handleClick' just pass `smoothScrollToTop` as a onClick function 💡